CHANGELOG.md

```diff
@@ -2,7 +2,8 @@
 
 - **[UPDATE]** New hooks for `MediaSizeProvider` and add a `mediaSizeForTestsOnly` override for test setup.
 - **[FIX]** Make `OnChangeParameters` part of the public API.
-[...]
+- **[NEW]** Add `SupportIcon`
```

Comment:
Hello @pith I couldn't find this icon on PIXAR LIbrary
https://www.figma.com/file/JjaALcWYWcqGtanufVLKYj/UI-%C2%B7-Pixar?node-id=3275%3A353

Where does it come from?